### PR TITLE
cap-dbt changes - squashed

### PR DESCRIPTION
**Pull request by J8cob1**
So I ended up doing this

### CODE REVIEW

**Comment by J8cob1** [comment]
There aren't really any new features with this commit, so the only testing that needs to be done is to make sure I didn't break any of the existing functionality. I think this would mainly mean mean making sure that the add/remove student and teams functionality still works, that the dashboard still loads data as expected, and that students reviews can be submitted as expected.

I did some minor testing, and most of everything seems to work adequately (I found a bug, but I seem to recall running into it before, so my code might not be the issue--I will note the bug in Trello)

File: capstone360/setup_mockup_db.py [hunk 1]
```diff
@@ -99,7 +98,7 @@ def generate_student_data():
                          + ".notreal@pdx.edu")
 
         # Get student id
-        student_id = id_num
+        student_id = str(id_num)
```
Comment (necrobuffalo) [inline capstone360/setup_mockup_db.py]: Do we need this cast?

Comment (J8cob1) [reply capstone360/setup_mockup_db.py]: No

File: capstone360/setup_mockup_db.py [hunk 2]
```diff
@@ -261,7 +228,7 @@ def fill_tables_with_data(cursor, student_data, num_sessions, num_teams):
 
             # Add student to the students table
             cursor.execute('INSERT INTO students VALUES(?,?,?,?,?,?,?,?)',
-                           (student_id,
+                           (str(student_id),
```
Comment (J8cob1) [inline capstone360/setup_mockup_db.py:231]: We might also not need this, but I figured it wouldn't hurt

**Review by necrobuffalo — APPROVED** [review]

**Comment by necrobuffalo** [comment]
Looks good but will wait to test and merge til morning.

**Pull request merged**